enterprise/internal/campaigns/syncer/syncer.go

```diff
@@ -452,6 +452,12 @@ func SyncChangeset(ctx context.Context, syncStore SyncStore, source repos.Change
 	if err := source.LoadChangeset(ctx, repoChangeset); err != nil {
 		_, ok := err.(repos.ChangesetNotFoundError)
 		if !ok {
+			// Store the error as the syncer error.
+			errMsg := fmt.Sprintf("Error while syncing changeset:\n\n```\n%s\n```", err.Error())
```

Comment:
We should probably add something 
```suggestion
			errMsg := fmt.Sprintf("Encountered error during last attempt to sync changeset data from code host:\n\n```\n%s\n```\nThis might be an ephemeral error that resolves itself at the next sync.", err.Error())
```